website/docs/cloud-docs/registry/publish-providers.mdx

```diff
@@ -20,72 +20,196 @@ Users must be members of an organization to access its registry and private prov
+
+```shell-session
+$ curl \
+  -T terraform-provider-aws_5.14.0_SHA256SUMS \
+  https://archivist.terraform.io/v1/object/dmF1b64hd73ghd63...
+```
+
+Next, upload the `SHA256SUMS.sig` file to the URL returned in the `shasums-sig-upload` field.
+
+```shell-session
+$ curl \
+  -T terraform-provider-aws_5.14.0_SHA256SUMS.72D7468F.sig \
+  https://archivist.terraform.io/v1/object/dmF1b37dj37dh33d...
+```
+
+### Create a provider platform endpoint
+
+First, calculate the SHA256 hash of the provider binary that you will upload. This should match the SHA256 hash of the file listed in the `SHA256SUMS` file.
+
+```shell-session
+$ shasum -a 256 terraform-provider-aws_5.14.0_linux_amd64.zip
+f1d83b3e5a29bae471f9841a4e0153eac5bccedbdece369e2f6186e9044db64e  terraform-provider-aws_5.14.0_linux_amd64.zip
+```
+
+Next, create a file named `platform.json`. Replace the `os`, `arch`, `filename`, and `shasum` fields with the values that match the provider that you will upload. 
+
+```json
+{
+  "data": {
+    "type": "registry-provider-version-platforms",
+    "attributes": {
+      "os": "linux",
+      "arch": "amd64",
+      "shasum": "f1d83b3e5a29bae471f9841a4e0153eac5bccedbdece369e2f6186e9044db64e",
+      "filename": "terraform-provider-aws_5.14.0_linux_amd64.zip"
+    }
+  }
+}
+```
+
+Use the [Create a Provider Platform endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-platform) to create a platform for the version. Platforms are binaries that allow the provider to run on a particular operating system and architecture combination (e.g., Linux and AMD64).  Replace `TOKEN` in the `Authorization` header with your Terraform Cloud API token and replace both instances of `ORG_NAME` with the name of your organization.
+
+```shell-session
+$ curl \
+  --header "Authorization: Bearer TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/organizations/ORG_NAME/registry-providers/private/ORG_NAME/aws/versions/5.14.0/platforms
+```
+
+The response includes a `provider-binary-upload` URL that you will use to upload the binary file for the platform.
+
+```json
+"links": {
+      "provider-binary-upload": "https://archivist.terraform.io/v1/object/dmF1b45c367djh45nj78"
+    }
+```
+
+### Upload provider binary
+
+Upload the platform binary file to the `provider-binary-upload` URL returned in the previous step. The example command below uploads the binary from your local machine.
+
+```shell-session
+$ curl -T local-example/terraform-provider-random_5.14.0_linux_amd64.zip
+  https://archivist.terraform.io/v1/object/dmF1b45c367djh45nj78
+```
+
+The version is available in the Terraform Cloud UI. Consumers can begin using this provider version in configurations. You can repeat these steps starting from the [Upload signatures](#upload-signatures) step to add additional platform binaries for the release.
```

Comment:
```suggestion
The version is available in the Terraform Cloud UI. Consumers can begin using this provider version in configurations. You can repeat these steps starting from the [Create a provider platform endpoint](#create-a-provider-platform-endpoint) step to add additional platform binaries for the release.
```

You don't have to upload new shasum/sig files, just make a new platform, upload the binary, for each combination of OS and arch you built support for.